packages/usa-in-page-navigation/src/styles/_usa-in-page-navigation.scss

```diff
@@ -0,0 +1,85 @@
+@use "uswds-core" as *;
+
+.usa-in-page-nav-container {
+  align-items: flex-start;
+  display: flex;
+  margin: units(8) units(0);
+
+  // Set main landmark max width to ensure in-page nav will fit
+  main,
+  [class^="main"] {
+    max-width: 87.5rem;
+    width: 100%;
+  }
+
+  .usa-in-page-nav {
+    display: none;
+    margin-left: units(0.5);
+    margin-top: units(2.5);
```

Comment:
@mitchmoccia Also curious why these margins are needed

Reply:
Since these determine the distance between the nav and both the top of the container and the text block, these should be settings to allow adaptation to individual projects

Reply:
Changed to settings